ui/imports/shared/stores/RootStore.qml

```diff
@@ -251,4 +251,29 @@ QtObject {
 
     property bool balanceHistoryIsLoading: Global.appIsReady? walletSectionAllTokens.balanceHistoryIsLoading : false
 
+    function getTokenCopy(modelData) {
```

Comment:
This is too fragile and ugly, sorry :) The code is slightly different now (due to https://github.com/status-im/status-desktop/pull/12953) but basically you can ask the SFPM model to get you the complete row of data you want, eg:

```cpp
    Q_INVOKABLE QVariantMap get(int row) const; // <-- all roles
    Q_INVOKABLE QVariant get(int row, const QString& roleName) const; // <-- one role
```

Reply:
I mean, using hardcoded role names like that will break the next time we add/change something in the model

Reply:
For the rationale, see the Note in:
https://doc.qt.io/qt-5/qml-qtquick-listview.html#delegate-prop